Client/src/Components/TabPanels/Account/ProfilePanel.jsx

```diff
@@ -223,7 +226,7 @@ const ProfilePanel = () => {
 				className="edit-profile-form"
 				noValidate
 				spellCheck="false"
-				gap={theme.spacing(20)}
+				gap={SPACING_GAP} // Applied SPACING_GAP for consistent spacing
```

Comment:
This is affecting the gap between groups of inputs, we also want to create spacing between label and input